aten/src/ATen/native/native_functions.yaml

```diff
@@ -3021,6 +3021,18 @@
     CPU: dense_to_sparse
     CUDA: dense_to_sparse
 
+- func: to_lazy(Tensor self) -> Tensor
+  variants: method
+  dispatch:
+    CPU: eager_to_lazy
+    CUDA: eager_to_lazy
+
+- func: to_eager(Tensor self) -> Tensor
+  variants: method
+  dispatch:
+    CPU: lazy_to_eager 
+    CUDA: lazy_to_eager 
```

Comment:
I know in earlier conversation, @zdevito has objected to API creep here. I personally think these functions are probably useful as a debugging mechanism, but we probably shouldn't make them methods (and give them suitably private sounding names).